src/validation/decorators/is_kolibri_trigger_n.ts

```diff
@@ -21,24 +21,22 @@ import {
     ValidatorConstraint,
     ValidatorConstraintInterface,
     ValidationArguments,
-    isBoolean,
-    isString,
     isNumber
 } from 'class-validator';
 
 
+export function isKolibriNodeTriggerN(value: any) {
+    return isNumber(value);
```

Comment:
Why is it not restricted like value ' and (value) <= 4' to make sure its a valid range?

Reply:
This is the validation for the TriggerN parameter, not the TriggerMode. TriggerN can by any number depending on the TriggerMode.